### PR TITLE
fix send button remains active in chats after sending reply containing a mention

### LINKED ISSUE #13770: Send button remains active in chats after sending reply containing a mention
**Steps:**

1. Open any chat
2. Send reply that contains a mention
3. Pay attention at Send button after sending a reply

**Actual result:** send button remains active

**Expected result:** send button is inactive 

https://user-images.githubusercontent.com/97245802/183241140-a21ce830-4357-4cee-af4d-ce57de5a0969.mp4

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/13770

### Testing Request:
Please test behavior of `send-button` with text & image, while typing and also after going back to home screen.

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/chat/components/input.cljs [hunk 1]
```diff
@@ -144,13 +144,9 @@
 
 (defn clear-input [chat-id refs]
   (hide-send refs)
-  (if (get @mentions-enabled chat-id)
-    (do
-      (swap! mentions-enabled dissoc chat-id)
-      ;;we need this timeout, because if we clear text input and first index was a mention object with blue color,
-      ;;after clearing text will be typed with this blue color, so we render white text first and then clear it
-      (js/setTimeout #(reset-input refs chat-id) 50))
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/chat/components/input.cljs]: hi @flexsurfer, Do we still need this timeout? I am not able to reproduce this issue anymore.

Comment (flexsurfer) [reply src/status_im/ui/screens/chat/components/input.cljs]: is it reason of the bug? i mean usually its better to keep such changes because who knows

Comment (Parveshdhull) [reply src/status_im/ui/screens/chat/components/input.cljs]: interestingly, this is not the cause, but still is fix :D

`hide-send` is disabling send button, but as we are calling `reset-input` delayed, rerender again enabling send button.
fixed enabling of `send-button`. Thank you

**Review by flexsurfer — APPROVED** [review]

**Comment by pavloburykh** [comment]
@Parveshdhull Thank you for the fix. Tested and ready for merge.

**Pull request merged**